### PR TITLE
feat: create 'badge' component and add a 'Beta' badge to sage300

### PR DESCRIPTION
**Pull request by JustARatherRidiculouslyLongUsername**
image

image

## Summary by CodeRabbit

- **New Features**
	- Introduced a customizable `BadgeComponent` that allows for dynamic styling based on size and theme.
	- Added a "Beta" badge next to the application name on the landing page for improved visibility.

- **Bug Fixes**
	- Enhanced visual spacing and layout of the landing page with updated HTML structure.

- **Documentation**
	- Created a new testing suite for the `BadgeComponent` to ensure reliability and performance.

- **Style**
	- Added SCSS styles for the badge component, supporting multiple themes and sizes.
	- Introduced new CSS custom properties for badge styling to enhance theme flexibility.

### CODE REVIEW

File: src/app/shared/components/core/badge/badge.component.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.badge {
+    display: inline-flex;
+    padding: var(--spacing-spacing-2, 2px) var(--spacing-spacing-6, 6px);
```
Comment (DhaaraniCIT) [inline src/app/shared/components/core/badge/badge.component.scss:3]: write the styles in tailwind form, like `tw-inline-flex tw-justify-center etc`

File: src/app/shared/components/core/badge/badge.component.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { Component, Input } from '@angular/core';
+
+
+type SizeOption = 'default' | 'large';
+type ThemeOption = 'brand' | 'light' | 'dark';
```
Comment (DhaaraniCIT) [inline src/app/shared/components/core/badge/badge.component.ts:5]: move `SizeOption` and `ThemeOption` as enum in enum.ts file. we should not define these in the component file.

File: src/app/shared/components/core/badge/badge.component.scss
Comment (DhaaraniCIT) [inline src/app/shared/components/core/badge/badge.component.scss:1]: write the styles in tailwind css formate, like `tw-inline-flex tw-justify-center etc` for the whole file

**Review by DhaaraniCIT — CHANGES_REQUESTED** [review]

**Review by ashwin1111 — APPROVED** [review]
take care of other comments in PR, pls also add screenshots in description for user visible change

**Review by ashwin1111 — APPROVED** [review]

File: src/app/shared/components/core/badge/badge.component.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.badge {
+    @apply tw-inline-flex tw-py-[2px] tw-px-[6px];
+    @apply tw-justify-center tw-items-center tw-rounded-full tw-text-center;
+
+    /* Body-semibold/font-10 */
+    @apply tw-text-[10px] tw-not-italic tw-font-500 tw-leading-[125%];
```
Comment (DhaaraniCIT) [inline src/app/shared/components/core/badge/badge.component.scss:6]: We have values for 10px, just do `tw-text-10-px`

**Pull request merged**